python/lsst/atmospec/centroiding.py

```diff
@@ -0,0 +1,173 @@
+#
```

Comment:
You should update this to match the preferred style: https://developer.lsst.io/python/style.html?highlight=python#each-python-file-must-contain-the-standard-license-preamble